server/db/config.js

```diff
@@ -62,7 +62,7 @@ knex.schema.hasTable('transactions').then((result) => {
       table.integer('item_id');
       table.integer('buyer_id'); // TODO: index
       table.integer('seller_id'); // TODO: index
-      table.string('order_status').defaultTo('In progress');
+      table.string('order_status').defaultTo('in progress');
```

Comment:
wrfwok that's fine thenyupyup yup yup yup yup yup yupsfghksdgsahgadwqwsdqwdqwdasfsgvaev qrf wf asffqwe ar asf asdgaqwrqwfqxdvsvsdfsagsdfgsadllllltltltltlltttttpppppthis is how the k\game works I think wow wowowowowowwowowoowowowwowowowo